### DISCUSSION TITLE
Hot Tub Time Machine

### DISCUSSION THREAD

**Discussion by electricityforprogress** [joeycastillo/Sensor-Watch#56] [Ideas]
I recently setup my dream  hot tub, and almost immediately i over did it between too hot temperature and too long of a soak.  This could be the perfect application for the temperature sensor and an alarm timer (sound and lights) in 15 minute increments. 

My big question is, would the hot tub be too adverse of a 'wet' environment for safe SensorWatch usage?  

I still need to spend time playing with the watch emulator and squeeze some of these ideas into code ;)  I'll update the thread with progress...

Thanks again joey for this great project, and your excellent updates!

**Comment by joeycastillo** [comment]
I get the sense that it would work in a hot tub environment! To an extent, Sensor Watch inherits some of the water resistance of the original F-91W, which I’ve seen described as being 30M water resistant. 

Sensor Watch reuses the same enclosure and gasket and requires no modifications to the watch case, so I sense that if you’re careful with the disassembly and reassembly, it’ll remain as such. I’ve definitely submerged the watch post-board-swap, showered with it and run it through the washing machine, and it’s worked fine after each test. I might not take it diving, but I sense that especially if you’re careful with the gasket when doing the swap, it wouldn’t be too fazed by a hot tub.

**Reply by electricityforprogress** [reply]
Thank you Joey.  Have you tried any destructive testing or 'catastrophic failure' of the hardware?   If the circuit does get wet do you think it is blue smoke city, or just throw the watch in a bag of rice overnight? ;)

The F-91W does fine in the hot tub on its own (so far).
